exercises/concept/interest-is-interesting/.docs/instructions.md

```diff
@@ -5,7 +5,7 @@ Each year, the balance of your savings account is updated based on its interest
 The interest rate your bank gives you depends on the amount of money in your account (its balance):
 
 - 3.213% for a negative balance (balance gets more negative).
-- 0.5% for a positive balance less than `1000` dollars.
+- 0.5% for a non negative balance less than `1000` dollars.
```

Comment:
```suggestion
- 0.5% for a positive balance greater or equal to 0 and less than `1000` dollars.
```

I actually like your second suggestion better so I applied it here.

Saying "non negative" for me doesn't fully solve the issue because while technically correct, it's still not explicit about 0.

Reply:
Good idea! The more explicit, the better. :+1: